src/main/java/seedu/address/logic/parser/DeleteEventCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.DeleteEventCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parse input index supplied and creates new DeleteEventCommand object
+ */
+public class DeleteEventCommandParser {
+
+    /**
+     * Parses the given String of arguments in the required format by the DeleteEventCommand
+     * and returns a DeleteEventCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteEventCommand parse(String args) throws ParseException {
+        try {
+            Index index = ParserUtil.parseIndex(args);
+            return new DeleteEventCommand(index);
+        } catch (ParseException e) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteEventCommand.MESSAGE_USAGE), e
```

Comment:
```suggestion
        } catch (ParseException pe) {
            throw new ParseException(
                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteEventCommand.MESSAGE_USAGE), pe
```
Might want to standardise to use `pe` instead of `e` for parse exception so that the code is cleaner and easier to maintain.

Reply:
ok